gemfire-junit/src/main/java/com/gemstone/gemfire/internal/cache/lru/LRUClockTest.java

```diff
@@ -366,6 +366,11 @@ public int getState() {
       return 0;
     }
 
+    @Override
+    public Object getValue() {
+      return null;
+    }
```

Comment:
I think for the purpose of this test, it should return false for "isValueNull()"